README.md

```diff
@@ -3,27 +3,53 @@
 
 # Rusk
 
-The Dusk's Smart Contract Platform.
+The official [Dusk](https://dusk.network/) protocol node client and smart contract platform.
 
 _Unstable_ : No guarantees can be made regarding the API stability, the project
 is in development.
 
+## Prerequisites
+
+- Rust 1.71 nightly or higher
+- GCC 13 or higher
+- Clang 16 or higher
+
+## Specification Requirements
+
+### Minimum Specifications
+
+| CPU | RAM | Storage | Network Connection |
+| :--- | :--- | :--- | :--- |
+| 2 cores; 2 GHz | 1 GB | 60 GB | 1 Mbps |
+
+### Recommended Specifications
+
+| CPU | RAM | Storage | Network Connection |
+| :--- | :--- | :--- | :--- |
+| 4 cores; 2 GHz | 4 GB | 250 GB | 10 Mbps |
+
 ## Build and Tests
 
-To run tests:
+To build `rusk` from source, Rust, GCC and Clang are required. Once the dependencies are installed, you can simply run the following command to compile everything:
 
+```bash
+make
```

Comment:
.env should be loaded before `make`
That because make keys requires it, and the keys are required to build circuits...

Reply:
Added it to all sections that indirectly call `make keys`, for consistency. Tnx!